src/rtnl/link/nlas/link_infos.rs

```diff
@@ -1226,6 +1245,151 @@ impl<'a, T: AsRef<[u8]> + ?Sized> Parseable<NlaBuffer<&'a T>> for InfoIpVlan {
     }
 }
 
+#[derive(Debug, PartialEq, Eq, Clone)]
+#[non_exhaustive]
+pub enum InfoMacSec {
+    Unspec(Vec<u8>),
+    Sci(u64),
+    Port(u16),
+    IcvLen(u8),
+    CipherSuite(u64),
```

Comment:
The kernel has
```c
/* cipher IDs as per IEEE802.1AE-2018 (Table 14-1) */
#define MACSEC_CIPHER_ID_GCM_AES_128 0x0080C20001000001ULL
#define MACSEC_CIPHER_ID_GCM_AES_256 0x0080C20001000002ULL
#define MACSEC_CIPHER_ID_GCM_AES_XPN_128 0x0080C20001000003ULL
#define MACSEC_CIPHER_ID_GCM_AES_XPN_256 0x0080C20001000004ULL


/* deprecated cipher ID for GCM-AES-128 */
#define MACSEC_DEFAULT_CIPHER_ID     0x0080020001000001ULL
#define MACSEC_DEFAULT_CIPHER_ALT    MACSEC_CIPHER_ID_GCM_AES_128
```


Can we use `enum MacsecCipherId` for them instead of u64?

Reply:
Please also implement `From<MacsecCipherId> for u64` and `From<u64> for <MacsecCipherId>` also.

Reply:
Done, thank you!